docs/gateway.md

```diff
@@ -51,6 +51,15 @@ Scarf Gateway configuration for a file package entry has three main consideratio
 
 See [Figure 3](#figure_3) to see how these pieces fit together visually.
 
+#### Python Packages
+
+Scarf Gateway configuration for a python package entry has three main considerations:
+
+- **pip Command**: This is the pip command used to install your package. This could be `pip install numpy` or an install command passing `--extra-index-url` if your package is not hosted on PyPI.org. This defines the location where the users will be redirected to when installing your package.
```

Comment:
```suggestion
- **pip Command**: This is the current pip command used to install your package. For packages on PyPI.org, this will be of the form `pip install numpy` and will include the `--extra-index-url` if your package is hosted elsewhere. This defines the location where the users will be redirected to when installing your package.
```